pymlg/jax/se23.py

```diff
@@ -194,6 +194,17 @@ def adjoint(X):
         Ad = Ad.at[6:9, 6:9].set(C)
         return Ad
 
+    @staticmethod
+    @jit
+    def adjoint_algebra(Xi):
+        A = jnp.zeros((9, 9))
+        A[0:3, 0:3] = Xi[0:3, 0:3]
```

Comment:
unfortunately jax arrays dont allow this kind of indexing, you have to do something more similar to the above